CHANGELOG.md

```diff
@@ -74,10 +74,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added a `timeout` argument to `DDPStrategy`. ([#13244](https://github.com/PyTorchLightning/pytorch-lightning/pull/13244))
 
-
-- Added `XLAEnvironment` cluster environment plugin ([#11330](https://github.com/PyTorchLightning/pytorch-lightning/pull/11330))
+- Added a `timeout` argument to `DDPSpawnStrategy`. ([#13244](https://github.com/Lightning-AI/lightning/pull/13383))
```

Comment:
```suggestion
- Added a `timeout` argument to `DDPSpawnStrategy`. ([#13383](https://github.com/Lightning-AI/lightning/pull/13383))
```

If you want, you could also combine it into one entry.

Reply:
Merge two change two entries into one